config/robots.rb

```diff
@@ -4066,4 +4066,191 @@
         }
       }
     )
+
```

Comment:
Is the BGE robots ready to be released?

Reply:
BGE isn't live yet, so the question is whether it's a problem having bed verification config present when the purposes don't yet exist. Think the answer must be that it's OK - this must be how we normally do it for work in progress pipelines. If it deploys to training OK and passes the Int Suite (for non-BGE tests) then I think it's fine.